src/arch/host/configs/library_defconfig

```diff
@@ -16,6 +17,7 @@ CONFIG_COMP_VOLUME=y
 CONFIG_COMP_VOLUME_LINEAR_RAMP=y
 CONFIG_COMP_VOLUME_WINDOWS_FADE=y
 CONFIG_DEBUG_MEMORY_USAGE_SCAN=n
+CONFIG_GOOGLE_RTC_AUDIO_PROCESSING_MOCK=y
```

Comment:
you can just enable the stub config

Reply:
Thanks, helps with two other conversions also, done!